docs/articles/nunit/writing-tests/ListMapper.md

```diff
@@ -13,9 +13,9 @@ shows two forms of the same assert.
 string[] strings = new string[] { "a", "ab", "abc" };
 int[] lengths = new int[] { 1, 2, 3 };
 
-Assert.That(List.Map(strings).Property("Length"),
+Assert.That(List.Map(strings).Property(nameof(string.Length)),
```

Comment:
I'd like to revert the instances in this file, as I think here it truly does make the understanding a little less clear at first glance.

Reply:
I can see both ways - for me personally, it's _nice_ that this reinforces the idea that "Length is a property on a string". I won't let it hold up anything - and I suppose my concern is partially remedied by the fact that the preceding method is `.Property(string property)`.